### ISSUE TITLE
Feature request: click turning white black depending on side is a nice touch

### ISSUE THREAD

**Issue by ijm8710** [lichess-org/mobile#822]
![image](https://github.com/lichess-org/mobile/assets/25938701/11df1b0f-410d-4633-bdef-712e796570c4)

**Comment by EmmetSchuler** [comment]
I believe this is already in the app? When playing games the clock is either white / black depending on whose move it is to play.

**Comment by ijm8710** [comment]
Yes I'm saying it's more intuitive to have it black/white based on what side you're on. They then slightly grey out/put into focus whether it's your turn. If you try on chess.com it's a design that makes a bunch more sense

**Comment by tom-anders** [comment]
Wouldn't it be better to be consistent with the lichess web UI instead here instead of imitating chess.com? So the side to move has a green clock, the other side grey?

**Comment by ijm8710** [comment]
While I agree parity is generally important, I feel the benefit trumps that.

Having the clock reflect what piece side you are adds a good bit of clarity, especially for an observer in a late but unusual endgame.
